example/lib/scanner_button_widgets.dart

```diff
@@ -166,10 +167,13 @@ class ToggleFlashlightButton extends StatelessWidget {
               },
             );
           case TorchState.unavailable:
-            return const Icon(
-              Icons.no_flash,
-              color: Colors.grey,
-            );
+            return const SizedBox(
```

Comment:
Could you add the missing trailing comma and wrap this in a Center widget?

```suggestion
            return const SizedBox(
               width: 48.0,
               child: Center(
                  child: Icon(
                     Icons.no_flash,
                     size: 32.0,
                     color: Colors.grey,
                  ),
               ),
            );
```